### ISSUE TITLE
Question: is there an option to turn on windows 10 sdk installer?

### ISSUE THREAD

**Issue by timmywil** [felixrieseberg/windows-build-tools#26]
**tldr;**
The README says that the windows 10 sdk is optional and off by default, but I don't see any documentation on how to turn that on. Is there a command line option?

_Long version_
I've come to the conclusion I need the windows 10 sdk because I've been debugging this for a couple days. My specific issue is with installing [bignum](https://www.npmjs.com/package/bignum) on Windows 10 64-bit. I've solved several issues before this and went so far as to do the "Option 2" installations from [here](https://github.com/Microsoft/nodejs-guidelines/blob/master/windows-environment.md#compiling-native-addon-modules). That worked, but I uninstalled all that to see if I can get it working with windows-build-tools. I _think_ the only substantial difference is that I had installed windows 10 SDK, and I believe that includes some OpenSSL tools which should fix it.

**So, how do I install Windows 10 SDK with windows-build-tools?**

The error I'm dealing with is...

```
LINK : fatal error LNK1181: cannot open input file 'C:\OpenSSL-Win64\lib\libeay32.lib' [C:\Users\shear\Code\opentable\start-page-nodejs\node_modules\ot-abtest-client\node_modules\bignum\build\bignum.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\timmy\AppData\Roaming\nvm\v4.4.4\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 10.0.14393
```

**Comment by timmywil** [comment]
Actually, nevermind. I don't know the difference. I tried installing the Windows 10 SDK separately, then tried installing OpenSSL and nothing worked.

**Comment by felixrieseberg** [comment]
Either way though, this would be a good one to have. @A92hm any chance you'd be interested in adding that? You know, for our glorious Microsoft?

The way this could work is that in addition to the current `post-install` step, we also throw in a little CLI - that way people could add/remove packages after installation.

**Comment by timmywil** [comment]
By the way, after further debugging, I figured out that while I *did* need the Windows 10 SDK, I *also* needed to install OpenSSL separately. I didn't think that was it when I posted this issue because I had tried it before. But the problem was the OpenSSL version. I posted an issue on [bignum](https://github.com/justmoon/node-bignum/issues/85) about my experience.

**Comment by felixrieseberg** [comment]
Merging this into https://github.com/felixrieseberg/windows-build-tools/issues/26

**Comment by nrakochy** [comment]
I am not sure that I understand the conclusion of this conversation. Is there a way to install Windows 10 sdk via the command line? 

I was hoping to see if this library sufficiently captured the [Windows React Native requirements](https://github.com/Microsoft/react-native-windows#requirements) without having to install VS. 

As an aside, it looks like the Windows 10 sdk "supports building desktop apps on Windows 8.1" ([source](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)), so perhaps the Windows 10 sdk would be a suitable default for the installer rather than 8.1. 

Thanks for the great tool.
